src/libraries/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -218,11 +218,11 @@ private X509Certificate2 EnsurePrivateKey(X509Certificate certificate)
                 }
 
                 X509Certificate2Collection collectionEx;
-                string certHash = certEx.Thumbprint;
+                string certHash = certEx!.Thumbprint;
```

Comment:
If `MakeEx` returns null above, this will NRE. Is this a bug?

Reply:
It seems likely. Maybe somebody from @dotnet/ncl could provide context?